ERCS/erc-7687.md

```diff
@@ -0,0 +1,245 @@
+---
+eip: 7687
+title: AUTHCALL Deposit/Withdrawal Extension for ERC-4626 Tokenized Vaults
+description: Allow Deposits and Withdrawals from Tokenized Vaults using EIP-3074 wallet signing.
```

Comment:
Not sure if this will fix it
```suggestion
description: Allow Deposits and Withdrawals from Tokenized Vaults using [EIP-3074](https://eips.ethereum.org/EIPS/eip-3074) wallet signing.
```